flink-yarn-tests/src/test/java/org/apache/flink/yarn/YarnTestBase.java

```diff
@@ -480,7 +483,7 @@ private static void writeHDFSSiteConfigXML(Configuration coreSite, File targetFo
      */
     public static void ensureNoProhibitedStringInLogFiles(
             final String[] prohibited, final Pattern[] whitelisted) {
-        File cwd = new File("target/" + YARN_CONFIGURATION.get(TEST_CLUSTER_NAME_KEY));
+        File cwd = new File("target/test/data/" + YARN_CONFIGURATION.get(TEST_CLUSTER_NAME_KEY));
```

Comment:
```suggestion
        File cwd = new File("target/test/data/" + YARN_CONFIGURATION.get(TEST_CLUSTER_NAME_KEY));
```
```suggestion
        File cwd = new File(
               GenericTestUtils.getTestDir(),
               YARN_CONFIGURATION.get(TEST_CLUSTER_NAME_KEY));
```
Also applies to other usages of this path.